### PR TITLE
resolve issue 364

### PR DESCRIPTION
**Pull request by thehesiod**
resolves https://github.com/aio-libs/aiopg/issues/364

### CODE REVIEW

**Comment by thehesiod** [comment]
will need to re-work this to enforce one cursor per connection, is slightly more complicated

**Comment by thehesiod** [comment]
ok, it now enforces one cursor per connection

**Comment by vir-mir** [comment]
@thehesiod  you add tests?

**Comment by thehesiod** [comment]
ok added a specific test

**Comment by thehesiod** [comment]
@asvetlov anything left for this?

**Comment by thehesiod** [comment]
bumped to 0.14

**Comment by thehesiod** [comment]
@vir-mir anything left to merge this?

File: aiopg/__init__.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
            'version', 'version_info', 'DEFAULT_TIMEOUT', 'IsolationLevel',
            'Transaction')
 
-__version__ = '0.15.0'
+__version__ = '0.16.0'
```
Comment (vir-mir) [inline aiopg/__init__.py]: no need to lift the version in a separate pull request

Comment (thehesiod) [reply aiopg/__init__.py]: did you mean: "no need to life the version`, it will be done ` in a separate pull request"?

Comment (vir-mir) [reply aiopg/__init__.py]: Yes, this is usually done release engineer, there may already be several tasks in the issue.

Comment (thehesiod) [reply aiopg/__init__.py]: ok, reverted

**Comment by vir-mir** [comment]
ok, merge into master

**Comment by smagafurov** [comment]
Can you bump version with this fix?

**Pull request merged**